package.json

```
@@ -22,6 +22,7 @@
   ],
   "devDependencies": {
     "@biomejs/biome": "1.6.4",
+    "@moonrepo/cli": "^1.23.4",
```

```suggestion
    "@moonrepo/cli": "1.23.4",
```

I personally don't like of installing `moon` this way, but I guess it's the best way for people not using it as default. We should at least pin the dependency though.

---

I personally dislike global dependencies (here if you install moon using pnpm) because there can be various versions between people, and you loose them when switching node versions

---

That's fine, we don't need to use global dependencies, just pin the version, and it's good.